packages/sales-api-service/src/services/permissions.service.js

```diff
@@ -31,9 +31,10 @@ export const generatePermissionNumber = async (
   const permit = await getReferenceDataForEntityAndId(Permit, permitId)
 
   const endDate = await calculateEndDateMoment({ permitId, startDate })
-  const endTime = moment(endDate)
-    .add(1, 'hour')
-    .startOf('hour')
+  const endTime =
+    permit.durationMagnitude === 12 && permit.durationDesignator.description === 'M'
+      ? moment(endDate)
+      : moment(endDate).add(1, 'hour').startOf('hour')
```

Comment:
I'm not sure we're testing both scenarios here - would you mind checking and adding a test / test case if we're not?

Reply:
@jaucourt From what I can see, both scenarios are covered as we have some tests which use a mock 12-month permit and some which use a mock 1-day permit. I could add some more if you'd prefer them to be more specific though.

Reply:
No, that's fine. Thanks for checking :)